lang/main.json

```diff
@@ -433,6 +433,11 @@
         "veryBad": "Very Bad",
         "veryGood": "Very Good"
     },
+    "filmstrip": {
+        "accessibilityLabel": {
+            "heading": "Video thumbnails"
+        }
+    },
```

Comment:
I didn't really know how to organize new i18n strings so I kinda copied the logic I already saw, is this ok?